doc(release): prepare release notes for MAP 22.10.4 (#2075)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-22.10/releases/centreon-commercial-extensions.md

## Centreon MAP

### 22.10.4

Release date : `soon`

#### Bug fixes

> - Fixed an issue preventing to create MAP with API

Reviewer: ```suggestion
- Fixed an issue preventing users to create a map with the API
```

> - Fixed an issue preventing users to create a map with the API

Reviewer 2: ```suggestion
- Fixed an issue preventing users to create a map with the API.
```

- Fixed an issue that caused server to fail to start when trying to load empty output.

> - Fixed links when bendpoints are empty cause map to become not editable.

Reviewer: ```suggestion
- Fixed issue where empty endpoints caused maps to become not editable.
```

> - Fixed display label and use ressource name properties for links.

Reviewer: ?

> - Fixed display label and use ressource name properties for links.

Reviewer: ```suggestion
- Resource name properties are now used for links when saving maps.
```
